tests/dummy/app/templates/event-propagation.hbs

```diff
@@ -0,0 +1,17 @@
+{{format-markdown "
```

Comment:
New docs page is very similar to the previous "Priority" but without things removed in 7.x